src/components/lime-ai-avatar/lime-ai-avatar.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, h, Host, Prop } from '@stencil/core';
+
+/**
+ * This component displays an avatar, representing Lime AI assistants.
+ *
+ * :::warning
+ * This is a private component used internally in the Lime's various applications,
+ * which is the reason for having it in Lime Elements —to ease the distribution
+ * of the component across all our apps.
+ *
+ * 3rd party developers are not allowed use this component directly.
+ * :::
+ *
+ * @private
+ * @exampleComponent limel-example-lime-ai-avatar-basic
+ * @exampleComponent limel-example-lime-ai-avatar-colors
+ */
+@Component({
+    tag: 'limel-lime-ai-avatar',
```

Comment:
Another question about the name then: Does it have to be `limel-lime-ai-avatar`? Would `limel-ai-avatar` not be enough? 🤔

I realise the idea is that this is the avatar of "Lime AI", but we don't even know if it's going to be called that.

Reply:
sure. or lime-ai-avatar? 😅 feel free to change. I'm ok with anything

Reply:
⚡